gateway/src/main/java/io/zeebe/gateway/impl/broker/cluster/BrokerClusterStateImpl.java

```diff
@@ -79,13 +79,10 @@ public void addPartitionIfAbsent(final int partitionId) {
   }
 
   public void addBrokerIfAbsent(final int nodeId) {
-    brokerAddresses.computeIfAbsent(
-        nodeId,
-        k -> {
-          brokerVersions.put(nodeId, "");
-          brokers.addInt(nodeId);
-          return "";
-        });
+    if (brokerAddresses.get(nodeId) == null) {
+      brokerAddresses.put(nodeId, "");
+      brokers.addInt(nodeId);
```

Comment:
Missing `brokerVersions.put(nodeId, "")`?

Reply:
Yup, missed it when rolling back. Thanks